### PR TITLE
Feat: Teacher Overview | Teacher Details 

### PR DESCRIPTION
**Pull request by ShutUpChips15**
- Functional Done 
- TODO CSS CRAP!

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { MatFormFieldModule } from '@angular/material/form-field';
 import { MatCardModule } from '@angular/material/card';
 import { MatInputModule } from '@angular/material/input';
```
Comment (jasonncleveland) [inline src/app/app.module.ts:6]: These imports are not alphabetical :disappointed:

File: src/app/teacher/group-detail/group-detail.component.ts [hunk 2]
```diff
@@ -7,9 +8,36 @@ import { Component, OnInit } from '@angular/core';
...
+  constructor(
+    private route: ActivatedRoute,
+    private router: Router
+  ) {
+  }
 
   ngOnInit(): void {
+    this.id = this.route.snapshot.params["id"];
+    this.items = [
+      { name: "Budget", id: 0,  subItems: [{ name: "L1: Income 1 attempt 2/10" }, { name: "L2: Fix Expenses 2 attempts 9/10" }, { name: "L3: Variable Expenses 3 attempts 10/10 " }] },
```
Comment (jasonncleveland) [inline src/app/teacher/group-detail/group-detail.component.ts:24]: Should be in a JSON file in `/assets`. Keeps the code cleaner.

File: src/app/teacher/group-detail/group-detail.component.ts [hunk 3]
```diff
@@ -7,9 +8,36 @@ import { Component, OnInit } from '@angular/core';
 })
 export class GroupDetailComponent implements OnInit {
 
-  constructor() { }
+  id: Number;
+  items: Object;
```
Comment (jasonncleveland) [inline src/app/teacher/group-detail/group-detail.component.ts:12]: TypeScript uses the types `number` for numerical and `any` for objects.

**Review by jasonncleveland — APPROVED** [review]

File: src/app/teacher/group-detail/group-detail.component.ts (hunk 2 above)
Comment (ShutUpChips15) [reply src/app/teacher/group-detail/group-detail.component.ts:24]: Don't have time to import, can do it tomorrow if we have time.

**Pull request merged**